### DISCUSSION TITLE
Como utilizar TabNavigator do React-Native?

### DISCUSSION THREAD

**Discussion by MatheusCoxxxta** [frontendbr/forum#1669] [Perguntas e Respostas]
Estou desenvolvendo um app e ele funciona de forma padrão, após efetuar o login o usuário é redirecionado para a Home do app.

Para criar o menu inferior, utilizei o **tabNavigator** do React-Native. Meu problema é o seguinte: após efetuar o login, em minha visão o usuário deve ser redirecionado para **"tabNavigator.js"**, que é o arquivo que gera o menu inferior e interliga as telas do app. Quando tento rodar isso, o celular trava, não consigo de jeito nenhum redirecionar o usuário de login para **"tabNavigator.js".** Agora estou em dúvida, **devo redireciona-lo para essa "tabNavigator.js" mesmo ou esse é meu erro? Qual a melhor forma de redirecionar o usuário do login para a Home page?**

**Comment by elfiservice** [comment]
Já usei, 

Cria o TabNaviagtor 
`const MainTabs = createBottomTabNavigator(routeTabConfig, TabNavigatorConfig)`

Ai Você cria uma StackNavigator e joga o MAinTabs dentro da stack junto com a pagina de login

```
const MainNavigator = createStackNavigator({
    login: {
      screen: Login,
      navigationOptions: {
        header: null,
      },
    },
    main: {
      screen: MainTabs,
    },

```

Agora é só navegar da Login para a Main
